jspdf.js

```diff
@@ -895,6 +983,35 @@ var jsPDF = (function(global) {
         // putHeader()
         out('%PDF-' + pdfVersion);
 
+        if (pdfVersion == '1.4') {
+            // add objects that were created using the newObject2 function
+            newObject2List.forEach(function (newObject2) {
```

Comment:
What is this newObject-stuff?

Reply:
These function workarounds for the current jsPDF architecture.  It allows for delayed rendering.